src/lib/mina_stdlib/time.ml

```diff
@@ -0,0 +1,24 @@
+open Core_kernel
+
+module Span = struct
+  [%%versioned
+  module Stable = struct
+    module V3 = struct
```

Comment:
If this is a new versioned type, why can't it start out at `V1`?

Reply:
The underlying type is time span V3. But you're probably right I should make this V1